### PR TITLE
Update GSoC-19.md

### PR DESCRIPTION
**Pull request by Mec-iS**

### CODE REVIEW

File: pages/GSoC-19.md [hunk 1]
```diff
@@ -4,5 +4,106 @@ title: Ideas page for GSoC 2019 | Hydraecosystem.org
...
+- [Graph Algorithms](https://en.wikipedia.org/wiki/Category:Graph_algorithms)
+
+**Difficulty Level** : *Intermediate/Hard*
+
+### Issues
+Some issues are already open for this idea, please [check this list](https://github.com/HTTP-APIs/python-hydra-agent/issues?q=is%3Aissue+is%3Aopen+label%3AGSOC-2019):
+
+## General Improvements in hydrus.
+### Idea:
+There si always room from improvement in our flagship server, hydrus:
```
Comment (PrajwalM2212) [inline pages/GSoC-19.md]: Speeling errors:  `si` should be `is` and `from` should be `for`

File: pages/GSoC-19.md [hunk 2]
```diff
@@ -4,5 +4,106 @@ title: Ideas page for GSoC 2019 | Hydraecosystem.org
...
+
+## Demonstration with Dynamic API paths
+### Idea:
+The objective is to create an API whose structure (paths to different kinds of data) is constantly changing, clients can still consume data by parsing the entrypoint vocabulary. Thanks to this it would be possible to use a Hydra client to discover the required paths for various kinds of data. This can be a great way to demonstrate the capabilities and use cases of HTTP-APIs and Hydra in general. We can have a UI showing the API structure in real-time and allow users to POST/GET/PUT/DELETE any type of data. We can also show how the client and API server interact with each other ( We had a lot of requests going on in the drone demo and it was very difficult to understand how things are working in the background). For example Suppose we have a Student class with basic properties like Name, Id, Class etc. Then the user can request for data say "Students with Id = 1*" without knowing anything about the API structure as it's dynamic. We can also demonstrate advanced querying features with this.
+### Skill Required:
+SkillsRequired :
```
Comment (xadahiya) [inline pages/GSoC-19.md]: Duplicate

File: pages/GSoC-19.md [hunk 3]
```diff
@@ -4,5 +4,106 @@ title: Ideas page for GSoC 2019 | Hydraecosystem.org
...
+
+**Difficulty Level** : *Intermediate/Hard*
+
+### Issues
+Some issues are already open for this idea, please [check this list](https://github.com/HTTP-APIs/python-hydra-agent/issues?q=is%3Aissue+is%3Aopen+label%3AGSOC-2019):
+
+## General Improvements in hydrus.
+### Idea:
+There si always room from improvement in our flagship server, hydrus:
+- implentation of a search mechanism (use of `hydra:search` and IRITmeplates, and possibly Triple Pattern Fragments)
```
Comment (PrajwalM2212) [inline pages/GSoC-19.md]: `implentation` should be `implementation`

**Comment by xadahiya** [comment]
Fixed typos, grammatical mistakes and moved to #13.

**Pull request closed without merging**